packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -514,6 +531,20 @@ export function TransferPanelMain({
           </BalancesContainer>
         </NetworkListboxPlusBalancesContainer>
       </NetworkContainer>
+
+      <Dialog
+        closeable
+        title='Token not supported'
```

Comment:
We use a tool called [Prettier](https://prettier.io) to automatically format code based on some predefined rules, so everyone on the team is in sync, and things like single or double quotes are handled automatically.

There's a VSCode extension: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode